www/pages/plan-trip/plan-trip-controller.js

```diff
@@ -443,6 +457,12 @@ angular.module('pvta.controllers').controller('PlanTripController', function ($s
     $scope.params.time.type = selectedOption.type;
   };
 
+  $scope.goToStop = function(loc) {
+    NearestStop.get({latitude: loc.lat(), longitude: loc.lng()}).$promise.then(function(stop) {
```

Comment:
Nope, but I have little idea how promises work so I don't really know what's happening there 😃  If you know a better way to write it then let me know (get.then causes an error)